lib/ansible/modules/crypto/luks_device.py

```diff
@@ -251,15 +317,21 @@ def is_luks(self, device):
 
     def run_luks_create(self, device, keyfile):
         # create a new luks container; use batch mode to auto confirm
-        result = self._run_command([self._cryptsetup_bin, 'luksFormat',
-                                    '-q', device, keyfile])
+        label = self._module.params.get('label')
+        command = [self._cryptsetup_bin, 'luksFormat', '-q', device, keyfile]
+        if label is not None:
+            # create luks container v2 with label
+            command = [self._cryptsetup_bin, 'luksFormat', '--type', 'luks2',
```

Comment:
Instead of replacing `command`, why not extend it? This way it is very hard to extend this code.

Reply:
Anyway, it's probably best to wait until #61603 is merged and then change this.

Reply:
I took your suggestion, thanks!